DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -212,7 +212,7 @@ private function loadJackalopeSession(array $session, ContainerBuilder $containe
         $backendParameters += $session['backend']['parameters'];
         // only set this default here when we know we are jackalope
         if (!isset($backendParameters['jackalope.check_login_on_server'])) {
-            $backendParameters['jackalope.check_login_on_server'] = $container->getParameter('kernel.debug');
+            $backendParameters['jackalope.check_login_on_server'] = false;
```

Comment:
these parameters are basically just piped to the phpcr implementation, the configuration class does not know about them. i don't see how i could set a default in the configuration class?